cli/azd/pkg/apphost/testdata/aspire-args.json

```diff
@@ -0,0 +1,32 @@
+{
+    "resources": {
+      "apiservice": {
+        "type": "project.v0",
+        "path": "../AspireArgs.ApiService/AspireArgs.ApiService.csproj",
+        "args": [
```

Comment:
FWIW - it wasn't obvious from the ACA console that these arguments were being passed (or at least I couldn't find the place in the UI that made this clear) but this does work end to end, I took Eric's idea of just writing the args to stdout in my project and then looking at the console log stream for the replica.